src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -238,6 +238,18 @@ Referenced assemblies:
 - 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 # Number of references: 26
 
+---
+Assembly: 'SonarLint.VisualStudio.IssueVisualization.Security, Version=4.28.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+Relative path: 'SonarLint.VisualStudio.IssueVisualization.Security.dll'
+
+Referenced assemblies:
+- 'Microsoft.VisualStudio.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
This is the version we don't want, isn't it?

Reply:
Yes, I'll remove it in the future PR